cmd/lokomotive-controller/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+	controller "github.com/kinvolk/lokomotive/pkg/controller"
+)
+
+const (
+	port = "8080"
+)
+
+var tlscert, tlskey string
+
+func usage() {
+	flag.PrintDefaults()
+	os.Exit(2) //nolint:gomnd
+}
+
+func main() { //nolint:funlen
+	flag.Usage = usage
+
+	err := flag.Set("logtostderr", "true")
+	if err != nil {
+		glog.Errorf("setting flag logtostderr failed: %v", err)
+
+		return
```

Comment:
`glog` also has a `Fatalf` func which does exactly this. This can be replaced in other places as well, in this file.

Reply:
Rather then using `Fatalf`, I'd move logic which may return error to separate function returning `error` and call `glog.Fatalf` only once, rather than using it for controling the flow. See #630.